### ISSUE TITLE
Request: Fish Shell

### ISSUE THREAD

**Issue by avarun42** [lukesampson/scoop#4220]
Would be awesome to have a package for `fish` shell. There's a package compiled for Windows maintained by [the folks over at MSYS2](https://packages.msys2.org/base/fish).

**Comment by HUMORCE** [comment]
This request has nothing to do with scoop.

but you can install fish shell in MSYS2, then call fish by command: `msys2 -c fish`

or create a function to do this, example:
```powershell
[HUMOR@W ~]$ function fish {msys2 -c fish}
[HUMOR@W ~]$ fish
Welcome to fish, the friendly interactive shell
Type `help` for instructions on how to use fish
HUMOR@W /c/U/HUMOR> echo $FISH_VERSION
3.1.2
```

define fish function in powershell $PROFILE, then you can call fish shell anywhere.

**Comment by bridystone** [comment]
Hi Humorce,

i don't understand the answer. i also would like to have fish installed on my Windows machine. 
i also thought that scoop could be a replacement of msys2 for fish and any other package out there.

is this a misunderstanding of the whole concept?

**Comment by tahir-hassan** [comment]
Yes, I also wanted `fish` to be added to `scoop`, but then after reading about POSIX etc, it seems as though it is not possible until the `fish` people compile a native version for Windows.

As the above respondents are describing, you can install and run `fish` within `msys2`. 

Therefore you would first install `msys2`:

```powershell
scoop install msys2
```

Then you run `msys2` (it may prompt you to re-run it twice), and then you use `msys2`'s own package manager `pacman` to install `fish`:

```bash
pacman -S fish
```

If you wish to have your `scoop` installed programs available while in MSYS,  you could add the following lines to your `~/.bashrc`:

PATH=$PATH:/c/Users/$USER/scoop/shims

Then run `fish` within `msys2`.
